src/Components/Components/src/RenderTree/ArrayBuilder.cs

```diff
@@ -15,26 +17,25 @@ namespace Microsoft.AspNetCore.Components.RenderTree
     /// components can be long-lived and re-render frequently, with the rendered size
     /// varying dramatically depending on the user's navigation in the app.
     /// </summary>
-    internal class ArrayBuilder<T>
+    internal class ArrayBuilder<T> : IDisposable
     {
-        private const int MinCapacity = 10;
+        // The following fields are memory mapped to the WASM client. Do not re-order or use auto-properties.
```

Comment:
```suggestion
        // The following fields are memory mapped to the WASM client. Do not re-order or use auto-properties, or add more fields, without also updating the corresponding TypeScript code.
```

Reply:
Actually ignore this. Adding fields wouldn't break it (if I remember correctly).